testing/btest/language/expire-func-type-check.zeek

```diff
@@ -12,3 +12,7 @@ global foo = 3;
 global t: table[string] of count &expire_func=myev;
 global tt: table[string] of count &expire_func=myhk;
 global ttt: table[string] of count &expire_func=foo;
+
+type ap: set[addr, port];
+global expire_ce: function(t: table[addr,port] of set[addr], s: ap): interval;
+global ce: table[addr, port] of set[addr]=table() &create_expire=1 secs &expire_func=expire_ce;
```

Comment:
Would it be worth adding a test here that does it correctly just to knowingly have a positive test case? I realize that there's lots of other tests that cover this, but it'd be good to have all of it in one place.

Reply:
Yeah, thanks, I'll make sure the test is more comprehensive.